apps/coa/fields/index.js

```diff
@@ -174,7 +174,14 @@ module.exports = {
 
   telephone: {
     mixin: 'input-text',
-    validate: ['ukPhoneNumber'],
+    validate: ['notUrl', 'internationalPhoneNumber'],
```

Comment:
As discussed, this requires a custom validation for the international number, as `internationalPhoneNumber` expects the country code to be provided as an additional attribute and does not allow spaces.